### DISCUSSION TITLE
「开源之道」主创接受社会赞助 - 开源之道

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#30] [Announcements]
# 「开源之道」主创接受社会赞助 - 开源之道

https://opensourceway.community/posts/the_way_of_open_source/kuosi-2022-find-sponsor/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)

发现随着持续阅读/交流, 过往文章也在持续修订...
[deletl company info for last article\. · OCselected/ttoos@0f58589](https://github.com/OCselected/ttoos/commit/0f5858970ee4928181493e1995366d013b7ba109)

![0f58589](https://ipic.zoomquiet.top/2022-05-14-zshot%202022-05-14%2020.11.00.jpg)

这其实是其它 blogger 们在以往发布平台/渠道中, 永远没有的...
以往都是基于 CMS, 现在我们是基于 版本管理系统;
所以, 习惯性将文章当成代码模块, 持续完善...

那么, 其实和当前在用的 giscus 类似, 复用 github 资源, 组合出全新公开功能;
这其实, 导致这种 blog 形式的网站结构, 并不真正合适这种主题讨论/图书创作基地;

值得在每篇文章尾部, 找到一种形式可以自动展示更新部分,
这样, 也是一种变更提醒, 也就是将网站的变化, 多出一种维度来展示...
